cluster-provision/k8s/1.26/manifests/kubeadm.conf

```diff
@@ -19,7 +19,11 @@ apiServer:
     audit-policy-file: /etc/kubernetes/audit/adv-audit.yaml
     enable-admission-plugins: NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota
     feature-gates: AdvancedAuditing=true
+    admission-control-config-file: /etc/kubernetes/psa.yaml
```

Comment:
Note please that there is a file named kubeadm_ipv6.conf
that should have the same changes

Reply:
Done, thank you for catching this.